acceptance/openstack/sharedfilesystems/v2/shareaccessrules_test.go

```diff
@@ -0,0 +1,48 @@
+//go:build acceptance
+// +build acceptance
+
+package v2
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud/acceptance/clients"
+	"github.com/gophercloud/gophercloud/acceptance/tools"
+	th "github.com/gophercloud/gophercloud/testhelper"
+)
+
+func TestShareAccessRulesGet(t *testing.T) {
+	client, err := clients.NewSharedFileSystemV2Client()
+	if err != nil {
+		t.Fatalf("Unable to create a shared file system client: %v", err)
+	}
+
+	client.Microversion = "2.49"
```

Comment:
this API should be available from "2.45": https://docs.openstack.org/api-ref/shared-file-system/?expanded=list-access-rules-deprecated-detail,list-shares-detail#share-access-rules-since-api-v2-45